flumine/order/order.py

```diff
@@ -391,7 +391,10 @@ def size_remaining(self):
         try:
             return self.current_order.size_remaining or 0.0
         except AttributeError:
-            return 0.0
+            if self.size_matched > 0:
```

Comment:
size_matched could be None? I think?

Reply:
I didn't think so based on https://github.com/liampauling/flumine/blob/master/flumine/order/order.py#L385 handling the potential None

Reply:
Good point, can you a small comment on this line as to why we do this? (placeResponse does not include sizeRemaining)

Reply:
Thanks, through some strategic print statements I was able to replicate your bug on v0.14.3 where size matched and size remaining both returned 0 when the order was in flight, this branch has fixed this 👍